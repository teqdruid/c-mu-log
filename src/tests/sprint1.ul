wall(4,4);
wall(6,3);
move("UP");
move("UP");
move("UP");
move("RIGHT");
move("RIGHT");
move("RIGHT");
face("foo");

face(symA);

main() {OR:
    @print("Wall: ", $x, ", ", $y);
    wall($x, $y);
    face($y);
    move($y);
}
