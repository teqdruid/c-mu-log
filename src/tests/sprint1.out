Wall: 4, 4

 ^^^  Solution  ^^^ 

Wall: 6, 3

 ^^^  Solution  ^^^ 

Wall: Any, 'foo'

 ^^^  Solution  ^^^ 

Wall: Any, symA

 ^^^  Solution  ^^^ 

Wall: Any, 'UP'

 ^^^  Solution  ^^^ 

Wall: Any, 'RIGHT'

 ^^^  Solution  ^^^ 

No more solutions
